tv-shows/src/components/core/ShowImage/ShowImage.tsx

```diff
@@ -13,7 +13,7 @@ export const ShowImage = ({ url, title }: IShowImage) => {
             src={imageUrl}
             alt={title}
             border='none'
-            borderTopRadius={10}
+            borderTopRadius='25px'
```

Comment:
Mislim da je common border za sve u dizajnu radius 24px, pa ga možda definirat u temi, a ovdje ga uključit kad "varijablu"?